src/ansys/motorcad/core/geometry.py

```diff
@@ -462,6 +484,123 @@ def find_entity_from_coordinates(self, coordinate_1, coordinate_2):
         return None
 
 
+class RegionMagnet(Region):
+    """Python representation of Motor-CAD magnet geometry region."""
+
+    def __init__(self, motorcad_instance=None):
+        """Initialise RegionMagnet."""
+        super().__init__(motorcad_instance)
+        self._magnet_angle = 0.0
+        self._mag_factor = 0.0
+        self._br_magnet = 0.0
+        self._magnet_polarity = ""
+
+    def _to_json(self):
+        """Convert from Python class to Json object.
+
+        Returns
+        -------
+        dict
+            Geometry region json representation
+        """
+        region_dict = super()._to_json()
+
+        region_dict["magnet_magfactor"] = self._mag_factor
+        region_dict["magnet_angle"] = self._magnet_angle
+
+        return region_dict
+
+    @property
+    def mag_factor(self):
+        """Get mag_factor.
```

Comment:
This and subsequent properties should be described in noun form. I'm not sure if ``mag_factor`` is equivalent to "Magnification factor" so please correct if this and later suggestions are wrong.

Reply:
Thanks for the comments @PipKat. Might have a better idea for this name. Will update now